app/src/main/resources/queries/query_for_item.rq

```diff
@@ -1,45 +1,26 @@
 SELECT
   ?item
-  (SAMPLE(?label) AS ?label)
-  (SAMPLE(?class) AS ?class)
-  (SAMPLE(?description) AS ?description)
-  (SAMPLE(?classLabel) AS ?classLabel)
-  (SAMPLE(?pic) AS ?pic)
-  (SAMPLE(?destroyed) AS ?destroyed)
-  (SAMPLE(?endTime) AS ?endTime)
-  (SAMPLE(?wikipediaArticle) AS ?wikipediaArticle)
-  (SAMPLE(?commonsArticle) AS ?commonsArticle)
-  (SAMPLE(?commonsCategory) AS ?commonsCategory)
-  (SAMPLE(?dateOfOfficialClosure) AS ?dateOfOfficialClosure)
-  (SAMPLE(?pointInTime) AS ?pointInTime)
+  ?itemLabel
+  ?itemDescription
+  ?class
+  ?classLabel
+  ?pic
+  ?destroyed
+  ?endTime
+  ?wikipediaArticle
+  ?commonsArticle
+  ?commonsCategory
+  ?dateOfOfficialClosure
+  ?pointInTime
 WHERE {
   SERVICE <https://query.wikidata.org/sparql> {
-    values ?item {
-      ${ENTITY}
-    }
+    VALUES ?item { ${ENTITY} }
   }
 
-  # Get the label in the preferred language of the user, or any other language if no label is available in that language.
-  OPTIONAL {?item rdfs:label ?itemLabelPreferredLanguage. FILTER (lang(?itemLabelPreferredLanguage) = "${LANG}")}
-  OPTIONAL {?item rdfs:label ?itemLabelAnyLanguage}
-  BIND(COALESCE(?itemLabelPreferredLanguage, ?itemLabelAnyLanguage, "?") as ?label)
+  # Get item label/class label/description in the preferred language of the user
+   SERVICE wikibase:label { bd:serviceParam wikibase:language "${LANG},en". }
```

Comment:
Let's say my phone is in French and I am visiting Japan.
So, ${LANG} is `fr` and most of the items around me only have labels in Japanese.
With this code, I do not see any label on most of the items, unfortunately (see screenshot).
 
 A solution for this can be to append all possible Wikidata languages to that string. I will increase maintenance a bit as language codes are added/removed/modified from time to time, but that's a reasonable solution.